src/standalone/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -431,9 +449,9 @@ function ClimaLSM.boundary_flux(
     Δz::ClimaCore.Fields.Field,
     Y::ClimaCore.Fields.FieldVector,
     p::NamedTuple,
-    t::FT,
-)::ClimaCore.Fields.Field where {FT}
-    return bc.bc(p, t) .+ ClimaCore.Fields.zeros(axes(Δz))
+    t,
+)::ClimaCore.Fields.Field
+    return FT.(bc.bc(p, t)) .+ ClimaCore.Fields.zeros(axes(Δz))
```

Comment:
same ?: do we need to convert to FT here, since it is added to a Field of (fixed) type? It might be converted automatically.